package.json

```diff
@@ -21,42 +21,40 @@
+    "postbuild:components": "node tasks/mod-extractor.js && prettier --cache --write components/*/metadata/*.md > /dev/null",
+    "build:preview": "nx run @spectrum-css/preview:build",
+    "build:site": "gulp build -LLL",
     "ci": "yarn build:all",
-    "ci:storybook": "nx storybook @spectrum-css/preview",
-    "clean": "yarn cache:clean && run-p clean:*",
-    "clean:components": "lerna exec --scope \"${SCOPE:-@spectrum-css/*}\" --ignore \"@spectrum-css/{*-builder*,preview,generator}\" -- rimraf dist",
-    "clean:docs": "rimraf dist",
-    "clean:preview": "nx clean @spectrum-css/preview",
-    "predev": "yarn build:components",
-    "dev": "NODE_ENV=development BROWSERSYNC_OPEN=true gulp dev",
-    "docs:mod": "node tasks/mod-extractor.js",
-    "postdocs:mod": "prettier --cache --write components/*/metadata/*.md > /dev/null",
-    "lint:components": "node ./tasks/packageLint.js",
-    "lint:styles": "stylelint",
-    "mv:preview": "rimraf dist/preview && mv tools/preview/storybook-static dist/preview",
+    "ci:storybook": "nx run @spectrum-css/preview:storybook",
+    "clean": "run-p clean:*",
+    "clean:cache": "nx reset",
+    "clean:components": "rimraf components/*/dist",
+    "clean:preview": "nx run @spectrum-css/preview:clean",
+    "clean:site": "rimraf dist",
+    "dev": "npm-run-all -s build:components -p dev:*",
+    "dev:components": "yarn watch:components",
+    "dev:site": "STORYBOOK_URL=\"http://localhost:3000\" gulp dev -LLL",
+    "dev:storybook": "yarn storybook --no-open --port 3000",
+    "lint": "run-p lint:*",
+    "lint:packages": "node ./tasks/packageLint.js",
+    "lint:styles": "stylelint \"components/**/*.css\" \"!components/*/dist/**\"",
+    "mv:preview": "rimraf dist/preview && cp -r tools/preview/storybook-static dist/preview || exit 0",
     "new": "nx run @spectrum-css/generator:new",
     "precommit": "lint-staged",
     "prepare": "husky install",
-    "prepare:site": "run-s clean:docs build:all",
-    "prerelease": "yarn version:build",
-    "release": "lerna publish",
-    "release:beta-from-package": "yarn release from-package --conventional-prerelease --preid beta --pre-dist-tag beta --no-private",
-    "prerelease:site": "yarn prepare:site",
-    "release:site": "gh-pages -d dist/ -f -e .",
-    "prestart": "yarn build:components",
-    "start": "NODE_ENV=development yarn workspace @spectrum-css/preview start",
-    "test": "nx test @spectrum-css/preview",
-    "test:scope": "nx test:scope @spectrum-css/preview",
-    "version:build": "lerna run --since origin/main build",
-    "watch": "gulp watch"
+    "release": "yarn build:components && lerna publish --no-private",
+    "release:beta": "yarn release:from-package --conventional-prerelease --preid beta --pre-dist-tag beta",
+    "release:from-package": "yarn build:components && lerna publish from-package --no-private",
+    "release:site": "yarn build:all && gh-pages -d dist/ -f -e .",
+    "start": "run-p storybook watch:components",
```

Comment:
Similar to my comment about running this command in isolation vs. running it as part of a series: if I run `yarn start` directly after `yarn clean`, Storybook attempts to start but fails due to missing `dist` assets. Maybe I shouldn't have been running it like that, but I wanted to be sure to mention it!

Running `yarn build:components` and then `yarn start` results in a successful Storybook launch and serve, but it looks like some styles assets may be missing:

<img width="742" alt="Screenshot 2023-08-22 at 1 52 08 PM" src="https://github.com/adobe/spectrum-css/assets/360251/da1d3f9f-84fe-45c2-980e-2a59524769e8">

Reply:
Added an update for this. So the challenge here is that storybook was compiling assets for itself but we want to start using the assets as they are compiled by the build now. That means updating the pathing from relative ../index.css references to "@spectrum-css/package-name" with alias's to the local directory.

Reply:
This should now be resolved